src/analysis/special_functions/trigonometric.lean

```diff
@@ -1244,6 +1271,15 @@ match lt_trichotomy x y with
 | or.inr (or.inr h) := absurd (tan_lt_tan_of_lt_of_lt_pi_div_two hy₁ hx₂ h) (by rw hxy; exact lt_irrefl _)
 end
 
+lemma has_deriv_at_tan_of_neg_pi_div_two_lt_of_lt_pi_div_two {x:ℝ} (h : x ∈ set.Ioo (-(π/2):ℝ) (π/2)) : has_deriv_at tan (1 / (cos x)^2) x :=
```

Comment:
These names seem excessively long (and the line is very long as well). Maybe something like `has_deriv_at_tan_of_small`?

Reply:
Or `has_deriv_at_tan_of_mem_Ioo`? We don't really need to encode the exact statement of the lemma in the name.

Reply:
I am more than happy to revise the names of those lemmas. In naming them I had adapted the nomenclature of the preexisting lemma `cos_pos_of_neg_pi_div_two_lt_of_lt_pi_div_two` so would you like me to change the name of that lemma as well?

Reply:
Yes, I think you can change that one too. There's no way to write these lemma names without tab complete anyway, so the end of the name isn't so important...

Reply:
Ok. I will try to come up with appropriate names for those lemmas. For the sake of convention I am reluctant to introduce names that have no precedent within the file.

Reply:
What about `has_deriv_at_tan_of_near_zero` ?